here-naksha-lib-core/src/test/java/com/here/naksha/lib/core/util/diff/PatcherTest.java

```diff
@@ -57,6 +56,48 @@ void basic() throws IOException {
     assertNull(newDiff);
   }
 
+  @Test
+  void testBasicNestedJson() {
```

Comment:
instead of multiple asserts, is it possible to use file based JSONAssert with strict checking? that will be easy to read and easy to maintain

Reply:
It's just been comparing Difference up till now, which we must confirm the class (InsertOp, UpdateOp, or RemoveOp). But yes I added a patch step in the test, which can be confirm with JSONAssert, though I use the alternative way, which is to use the Patcher, and make sure that the Difference between the patched feature, and the expected feature loaded from disk, is null